Feature/standardize principle formatting

amanda-wein opened the pull request:
## Summary of the discussion

Standardized the formatting of the principles, as discussed in https://github.com/OEOFoundry/OEOFoundry/discussions/10

## Type of change (CHANGELOG.md)

### Changed

- adjusted formatting of Principles Overview page
- adjusted formatting of Principle - Openness page
- adjusted formatting of Principle - Common Format page
- adjusted formatting of Principle - Textual Definitions page
- adjusted formatting of Principle - Versioning page

## Workflow checklist

### PR-Assignee

- [ ] 🐙 Follow the workflow in [CONTRIBUTING.md](https://github.com/OEOFoundry/OEOFoundry/blob/production/CONTRIBUTING.md)
- [ ] 📝 Update the [CHANGELOG.md](https://github.com/OEOFoundry/OEOFoundry/blob/develop/CHANGELOG.md)
- [ ] 📙 Update the documentation
- [ ] 🐙 Assign a reviewer to the PR

### Reviewer

- [ ] 🐙 Follow the [Reviewer Guidelines](https://github.com/OEOFoundry/OEOFoundry/blob/production/CONTRIBUTING.md#40--let-someone-else-review-your-pr)
- [ ] 🐙 Provided feedback and show sufficient appreciation for the work done

amanda-wein committed:
Update principles.md - add description of info sections

amanda-wein committed:
Update 5_version.md - standardized formatting

amanda-wein committed:
Update 8_definitions.md - standardized formatting

amanda-wein committed:
Update 2_format.md - standardized formatting

amanda-wein committed:
Update 1_open.md - standardized formatting

amanda-wein committed:
Update mkdocs.yml - fixed principle name typo

In the navigation for the Principles page, changed the principle name "Open" to "Openness" to be consistent throughout the website.

amanda-wein committed:
Update principles.md - added "naming conventions" to principle list

stap-m commented on docs/principles/1_open.md:
```diff
@@ -1,36 +1,27 @@
-- be released under a **public domain** mark or **permissive open license**
-- have an [OSI approved](https://opensource.org/licenses) or [open definition approved](https://opendefinition.org/licenses/api/) open license
+## Implementation
+- OEOF ontologies MUST
```
I know that I used OEOF as abbreviation before but right now, I'm not sure whether we agreed on this officially? 😅

stap-m replied:
Besides this question, this looks fine to me! Thanks @amanda-wein

amanda-wein replied:
No problem, I can go back and swap the abbreviation to "OEO Foundry" instead

stap-m replied:
Ok. But I this it is worth a discussion anyway: https://github.com/OEOFoundry/OEOFoundry/issues/37

amanda-wein committed:
remove OEOF abbreviation

amanda-wein committed:
remove OEOF abbreviation

amanda-wein committed:
remove OEOF abbreviation

amanda-wein committed:
remove OEOF abbreviation

amanda-wein committed:
Update 7_naming.md - standardized formatting

amanda-wein commented:
I saw that the naming convention principle has also been published on the website, so I adjusted the formatting on that one too and added it into this pull request
